build.sbt

```diff
@@ -65,13 +65,16 @@ lazy val schema =
 
 lazy val sequence = project
   .in(file("modules/sequence"))
-  .dependsOn(schema.jvm % "compile->test")
+  .dependsOn(schema.jvm)
```

Comment:
Ah, now I understand what this was for :) I think it was supposed to be `"compile;test->test"` i.e. "compile depends on compile, test depends on test", vs `"compile->test"` i.e. "compile depends on test".

But this works too.